addons/sales_team/views/crm_team_views.xml

```diff
@@ -18,6 +18,8 @@
             <menuitem action="base.action_partner_category_form" id="menu_partner_category_form" name="Contact Tags" sequence="1" parent="menu_config_address_book" groups="base.group_no_one"/>
             <menuitem action="base.action_partner_title_contact" id="menu_partner_title_contact" name="Contact Titles" parent="menu_config_address_book" sequence="3" groups="base.group_no_one"/>
             <menuitem id="menu_localisation" name="Localization" parent="menu_config_address_book" sequence="5" groups="base.group_no_one"/>
+            <menuitem id="res_partner_industry_menu" name="Sector of Activity" action="base.res_partner_industry_action" parent="sales_team.menu_config_address_book" sequence="7" groups="base.group_no_one"/> 
```

Comment:
"Sector**s**" should be plural. And perhaps we could move it a bit higher in the sequence order (e.g. 4) to avoid seeing it abandoned after the Localization sub-items ;-) 
![position](https://cloud.githubusercontent.com/assets/2039870/26259135/e1a9e2fe-3cc7-11e7-86c2-c2781f209f1c.png)